src/components/NavLayoutPage.js

```diff
@@ -5,9 +5,7 @@ const NavLayout = () => (
     <nav>
       <ul>
         <li><Link to="/">Books</Link></li>
-        <li>
-          <Link to="/categories">Categories</Link>
-        </li>
+        <li><Link to="/categories">Categories</Link></li>
       </ul>
     </nav>
     <Outlet />
```

Comment:
[Optional] You did well with your project, even though styling is not part of the requirement, it woud still be considerably good if your design follows the wireframe given or used so that when you have to start styliing you don't start reassembling your design afresh. This is because your framework is completely different from the design given 😄 

Your design :

![image](https://github.com/ctokoli/bookstore-with-react/assets/20537712/e441e54e-331c-40d2-aac1-9a6fffabec08)

Framework :
![Uploading image.png…]()